source/Junkership.js

```diff
@@ -5,18 +5,24 @@ import Reference from "./Reference.js"
 import Projectile from "./Projectile.js"
 import Score from "./Score.js"
 
-var junkerTex = Pixi.Texture.fromImage(require("./images/blue-starship.png"))
-
 export default class Junkership extends Pixi.Sprite {
     constructor() {
-        super(junkerTex)
+        super(PIXI.loader.resources.redJunkership.texture)
         game.playerCount++
         this.speed = 60
         this.score = new Score()
+
+        this.hitBox = new Pixi.Rectangle( this.x + 1 , this.y + 1 ,
+            this.width - 3 , this.height - 3 )
```

Comment:
We can, but it won't clean up the math. hitBox is a Rectangle, not a Sprite, so it -has- to be drawn from the top left. Changing the origin of the Junkership will make the math even uglier, IMO. Unless I'm misunderstanding your meaning or the mechanics.

I can, however, pull those out to declared constants to minimize magic numbers, if you think that would help.